tuf/repository_tool.py

```diff
@@ -991,7 +1000,7 @@ def add_signature(self, signature, mark_role_as_dirty=True):
 
 
 
-  def remove_signature(self, signature):
+  def remove_signature(self, signature, mark_role_as_dirty=True):
```

Comment:
Marking roles dirty when removing (or adding) signatures seems a bit odd, since the flag is used elsewhere to signal that the actual payload was modified and that it needs to be re-signed and written to disk.

While I can see how we somehow need to indicate that the signature removal (or addition) needs to be persisted on disk, this is different and also does not need to be propagated to any other metadata.